sdk/src/cache/cache_client_builder.rs

```diff
@@ -58,17 +62,44 @@ impl CacheClientBuilder<NeedsCredentialProvider> {
 }
 
 impl CacheClientBuilder<ReadyToBuild> {
+    pub fn with_num_connections(self, num_connections: u32) -> CacheClientBuilder<ReadyToBuild> {
```

Comment:
The usual type for a size is usize.

Reply:
This is a public-facing API, though... would a user want/expect to know that this is a size? As opposed to just knowing the range of integers that is valid? I don't mind changing it if you think it's the right thing to do but as a rust n00b `u32` seems like a fairly expected/reasonable type to see in this fn signature, while `usize` makes me raise my eyebrows and ask questions about implementation details.

Reply:
usize is commonly used for [container lengths](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.len) in rust, and a connection count sounds like a container limit. u32 is not a bad choice; it supports well and truly more than the range you need, it's just not what I'd expect to see.

Reply:
okay, i can change it to usize if you think that's what a rust dev would expect the API to surface.

Reply:
done in https://github.com/momentohq/client-sdk-rust/pull/393/commits/9612858e020df7ad4e5c1b6e602b756266efbd59